api/src/main/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivator.java

```diff
@@ -93,17 +93,17 @@ private void installConcepts() {
         	Context.flushSession(); //Flush so that purges are not deferred until after data import
         	
         	DataImporter dataImporter = Context.getRegisteredComponent("dataImporter", DataImporter.class);
-            dataImporter.importData("Reference_Application_Concepts-25.xml");     
+            dataImporter.importData("Reference_Application_Concepts-26.xml");     
             
-            if (ModuleUtil.compareVersion(OpenmrsConstants.OPENMRS_VERSION_SHORT, "2.2") < 0) {
-                dataImporter.importData("Reference_Application_Numeric_Concepts-25-pre2.x.xml");
+            if (ModuleUtil.compareVersion(OpenmrsConstants.OPENMRS_VERSION_SHORT, "2.3") < 0) {
```

Comment:
@sherrif10   , any reason for changing this to "2.3"    ??

Reply:
@mozzy11 That's according to the workflow,  since mdsbuilder is currently running on platform version 2.3.2, i think that version should be up to date with short version

Reply:
did you try to understand why we had to do this specifically for numeric concepts ??

Reply:
@mozzy11 , i haven't worked on adding numeric concept , this should be done after resolving errors related with dataexchange module